### DISCUSSION TITLE
Can't Figure Out How To Customize And Override Controller PLEASE HELP I'm Deseperate

### DISCUSSION THREAD

**Discussion by saidromadhon1** [uasoft-indonesia/badaso#459] [Q&A] [answered]
I've following the steps from documentations, but it doesn't work. Please Help

**Comment by rizkiheryandi** [comment, marked as answer]
Helo @saidromadhon1 , sorry for late response, forget to check the "discussion" tab

Could you show the error when doing the installation ?

So we can analyze it better

Btw, you can read the docs first https://badaso-docs.uatech.co.id/customization/overriding-controller

Thank you
